Agent/FeatureFlags/NRMAFlags.m

```diff
@@ -47,7 +47,8 @@ + (NRMAFeatureFlags) featureFlags
                               NRFeatureFlag_NetworkRequestEvents | 
                               NRFeatureFlag_RequestErrorEvents |
                               NRFeatureFlag_DistributedTracing |
-                              NRFeatureFlag_AppStartMetrics;
+                              NRFeatureFlag_AppStartMetrics |
+                              NRFeatureFlag_AutoCollectLogs;
```

Comment:
Careful with new feature flags: we _try_ to keep FF parity between the agents.

Reply:
I think it's very important to have this feature flag for iOS even if it's not on Android. This is because it breaks logging to Xcode console. If we want to keep parity I would greatly appriciate Android adding the feature flag and ability to disable auto capturing logs.

Reply:
Instead of a feature flag, could we detect when the debugger is attached? I imagine we wouldn't want to collect logs in that instance

Reply:
I think we still need the feature flag, and I believe this auto capture should be disabled by default.  It should also not collect logs when the debugger is attached.

Reply:
The reason being that we like to gate all new features with a feature flag in case there is issues.